test/e2e/framework.go

```diff
@@ -36,9 +36,11 @@ import (
 type Framework struct {
 	BaseName string
 
-	Namespace                *api.Namespace
-	Client                   *client.Client
-	NamespaceDeletionTimeout time.Duration
+	Namespace                         *api.Namespace
+	Client                            *client.Client
+	NamespaceDeletionTimeout          time.Duration
+	GatherKubeSystemResourceUsageData bool
```

Comment:
Please add empty line before this line and add some comment what exactly it does.
Does it print this data, does it verify it, etc.